eng/WpfArcadeSdk/tools/CreateTestPayload.targets

```diff
@@ -0,0 +1,49 @@
+<Project>
+    <!-- We use a local RID here since we don't want to change the build layout by setting the actual RID. -->
+  <PropertyGroup>
+    <LocalRuntimeIdentifier Condition="'$(Platform)'=='x64' or '$(Platform)'=='x86' or '$(Platform)'=='arm' or '$(Platform)'=='arm64'">win-$(Platform)</LocalRuntimeIdentifier>
```

Comment:
This is now at least 3 places (counting InjectModuleInitializer.targets) where we do this.  We should think about re-factoring this out into something common.